app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>DnStsOe26</title>
```

Comment:
Tìm hiểu và sử dụng `I18n cho TOÀN BỘ TEXT TRONG TẤT CẢ CÁC FILE HTML` NHÉ

Reply:
i18n tới phần static pages em add vô luôn nha chị?